confgenerator/logging_processors.go

```diff
@@ -229,3 +231,30 @@ var LegacyBuiltinProcessors = map[string]LoggingProcessor{
 		},
 	},
 }
+
+// A LoggingProcessorExcludeLogs filters out logs according to a pattern.
+type LoggingProcessorExcludeLogs struct {
+	ConfigComponent `yaml:",inline"`
+	MatchesAny      []string `yaml:"matches_any" validate:"required,dive,filter"`
```

Comment:
CC @sophieyfang as the decision of this name might influence the config design of the multiline support that you are working on.

Reply:
I prefer `match_any`. Bcause parse is a verb and we didn't add plural, and match is also a verb, to be consistent with what we have already, so `match_any`?

 @igorpeshansky who knows really well on naming conventions. There are cases we shall add plural and sometimes not. I can't remember the rules exactly...

Reply:
No reason for `matches_any`, I'm happy to make this change if it's more consistent.

Reply:
Let's do `match_by` for consistency then.

Reply:
Looks like it has been addressed. Resolving this thread.

Reply:
@sophieyfang I don't think it's a parallel because `parse_json` is a processor type, `match_any` is a config setting for the `exclude_logs` processor type.

I don't feel too strongly but I think my reasoning for `matches` was that it should be read as "exclude logs <if it> matches any of <these filters>".